repo2docker/buildpacks/base.py

```diff
@@ -14,7 +14,7 @@
 
 # Only use syntax features supported by Docker 17.09
 TEMPLATE = r"""
-FROM buildpack-deps:bionic
+FROM docker.io/buildpack-deps:bionic
```

Comment:
```suggestion
FROM docker.io/library/buildpack-deps:bionic
```
I thought `/library/` was always required when using the full name, but the tests seem to be passing without. However https://github.com/docker-library/official-images links to https://hub.docker.com/u/library